.eslintrc.js

```diff
@@ -78,6 +78,23 @@ module.exports = {
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
       })
-    }
+    }, {
+      // mocha tests
+      files: [
+        'node-tests/**/*.js'
+      ],
+      env: {
+        node: true,
+        mocha: true
+      }
+    }, {
+      // work-around unexpected (false positive?) eslint error
+      files: [
+        'blueprints/ember-electron/index.js'
+      ],
+      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+        "node/no-unpublished-require": "warn"
```

Comment:
@bendemboski Do you know why this is needed? It seems too coincidental with [the `require` problem I had to work-around](https://github.com/felixrieseberg/ember-electron/pull/354#issuecomment-403650859) in https://github.com/felixrieseberg/ember-electron/pull/355 to sit well with me

Reply:
Sure, so `no-unpublished-require` is meant to verify that the package doesn't try to use any code that isn't guaranteed to be there at runtime, e.g. accidentally adding something `require`d at runtime to `devDependencies` instead of `dependencies`. But it's 100% safe to assume that when a blueprint is running, `ember-cli` is present because `ember-cli` is what actually executes the blueprints.

This is actually very closely analogous to the use case described [here](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-require.md#allowmodules), so I think it would be best to leave it as an error, but specify `ember-cli` under `allowModules` here.

Reply:
Next question: can we easily apply this only to the `ember-cli` so we still get the benefit of the check in cases where it could actually be a problem? Should we just use a single-line comment in this one spot?

Reply:
👍